docs/modules/ROOT/pages/how-tos/configuring/customizing-the-build.adoc

```diff
@@ -176,6 +176,103 @@ An example of a custom task added to the pipeline that sends a slack notificatio
+[NOTE]
+====
+You can find more tasks link:https://quay.io/organization/konflux-ci[here]. Task generating Trusted Artifacts have `-oci-ta` suffix.
+====
+
+.Adding oci-ta task to pipeline 
+
+In your Pipeline definition, add a new task to the `tasks` section.
+
+[source,yaml]
+--
+spec:
+  tasks:
+    # ...
+    - name: clone-repository-oci-ta
+      params:
+      - name: url
+        value: $(params.git-url)
+      - name: revision
+        value: $(params.revision)
+      - name: ociStorage
+        # needs to be unique storage name
+        value: $(params.output-image).git
+      runAfter:
+      - init
+      taskRef:
+        params:
+        - name: name
+          value: git-clone-oci-ta
+        - name: bundle
+          value: quay.io/konflux-ci/tekton-catalog/task-git-clone-oci-ta@sha256:0f4360ce144d46171ebd2e8f4d4575539a0600e02208ba5fc9beeb2c27ddfd4c
+        - name: kind
+          value: task
+        resolver: bundles
+      workspaces:
+      # use the git-auth workspace for credentials
+      - name: basic-auth
+        workspace: git-auth  
+--
+
+The trusted tasks generate `SOURCE_ARTIFACT` result. This result can be used by other tasks in the pipeline.
+
+.Using `SOURCE_ARTIFACT` in another task
+
+[source,yaml]
+--
+spec:
+  tasks:
+    # ...
+    - name: task-using-trusted-artifact
+      # make sure the task runs after the artifact is created
+      runAfter:
+        - clone-repository-oci-ta
+      params:
+        # store the trusted artifact
+        - name: SOURCE_ARTIFACT
+          value: $(tasks.clone-repository-oci-ta.results.SOURCE_ARTIFACT)
+      taskSpec:
+        params:
+          - description: The Trusted Artifact URI pointing to the artifact with the application source code.
+            name: SOURCE_ARTIFACT
+            type: string
+        volumes:
+          # New volume to store a copy of the source code accessible only to this Task.
+          - name: workdir
+            emptyDir: {}
+
+        stepTemplate:
+          volumeMounts:
+            - mountPath: /var/workdir
+              name: workdir
+        steps:
+        # Add the trusted artifact to the task volume
+        - name: use-trusted-artifact
+          image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:8391272c4e5011120e9e7fee2c1f339e9405366110bf239dadcbc21e953ce099
+          args:
+            - use
+            - $(params.SOURCE_ARTIFACT)=/var/workdir/source
+        - name: task-run
+          # set the working directory to value from previous step
+          workingDir: /var/workdir/source
+          # Use image that suites your use case
+          image: registry.access.redhat.com/ubi8/minimal
+          securityContext:
+          # If the task step needs write access to the volume, set the runAsUser to 0 (root). 
+            runAsUser: 0
+          script: |
+            #!/bin/bash
+            # Execute the task
```

Comment:
This ends up being a lot of rendered yaml content. Would it be better to just reference an example in _some_ repository that has the content annotated, to reference some annotated "sample" task definition that has the required steps, or to just have smaller snippets of code isolated to the specific requirements?

Reply:
We have a sample here: https://github.com/RedHatInsights/chrome-service-backend/blob/main/.tekton/chrome-service-pull-request.yaml#L476-L514

If you prefer linking to an existing example I ok with that.